src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,101 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.model.patient.comparators.NameComparator.NAME_COMPARATOR;
+import static seedu.address.model.patient.comparators.PreferredNameComparator.PREFERRED_NAME_COMPARATOR;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.model.Model;
+import seedu.address.model.patient.Patient;
+
+/**
+ * Sorts (based on specified comparator) and list all patients in the address book to the user.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_SORT_SUCCESS = "Successfully sorted all patients by ";
```

Comment:
Maybe follow the way the other commands do it, which is to use `%1$s` and `String.format()`

Reply:
Good catch! Will proceed to change it!